src/test/java/com/google/api/codegen/util/CommonRenderingUtilTest.java

```diff
@@ -36,4 +36,15 @@ public void testStripQuotes() {
     assertThat(CommonRenderingUtil.stripQuotes("'a'bc'")).isEqualTo("'a'bc'");
     assertThat(CommonRenderingUtil.stripQuotes("\"a\"bc\"")).isEqualTo("\"a\"bc\"");
   }
+
+  @Test
+  public void testGetDocLines() {
+    // Check that we don't care which form of line break is used.
+    assertThat(CommonRenderingUtil.getDocLines("a\nb\nc")).containsExactly("a", "b", "c").inOrder();
+    assertThat(CommonRenderingUtil.getDocLines("a\rb\rc")).containsExactly("a", "b", "c").inOrder();
```

Comment:
Do we actually want to split on `\r`? I thought that the "newline sequence" was only `\r\n` on windows.

Reply:
It is, but my understanding was that Macs used \r. It would be a shame to get it working for Windows but not Mac. (For example, BufferedReader breaks on all three options.)